### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by garussell**
### Description of changes

Added first draft for functional README.

### Test Suite
- [x] Are all tests within the test suite passing?

### Specific Feedback Request(s)

Please let me know if you have any feedback or suggestions.

### CODE REVIEW

**Review by JorjaF — CHANGES_REQUESTED** [review]
Hey there are a couple of things that we wanted to tweek a bit, but this looks really great. thank you for setting that up/

File: README.md [hunk 1]
```diff
@@ -1,24 +1,84 @@
...
 
-Things you may want to cover:
+1. [Project Description](#project-description)
+2. [Inspiration](#inspiration)
+3. [Testamonials](#testamonials)
+4. [Case Studies](#case-studies)
+5. [Features](#features)
+6. [Usage](#usage)
+7. [Future Iterations](#future-iterations)
+8. [Contact](#contact)
```
[the remark this answers is not in the document]
Comment (garussell) [reply README.md:15]: Done

**Review by garussell — COMMENTED** [review]
Requested changes complete.

**Review by kcockett — APPROVED** [review]
Changes look good to me

**Review by kcockett — APPROVED** [review]
Changes were completed.  This looks good.

**Pull request merged**